pkg/kubelet/cm/types.go

```diff
@@ -31,6 +31,8 @@ type ResourceConfig struct {
 	CpuQuota *int64
 	// CPU quota period.
 	CpuPeriod *int64
+	// CPU sets. Which cpu to use
```

Comment:
Link to CPU List format: http://man7.org/linux/man-pages/man7/cpuset.7.html#FORMATS